### ISSUE TITLE
Shade filter options and round out corners 

### ISSUE THREAD

**Issue by annieduong** [trycrmr/covid-data-discovery#97]
Consider shading filter options a different color and rounding out the boxes so it looks more modern.

**Comment by trycrmr** [comment]
Indeed. Is the look of the "Show Menu" button adequate? Making the button design consistent makes sense.
